DOCSUP-3537: Documented the output_format_pretty_row_numbers setting (#16446)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

└───────────────┘
```

## output_format_pretty_row_numbers {#output_format_pretty_row_numbers}

> Enables adding row numbers before each row in [Pretty](../../interfaces/formats.md#pretty) output format.

Reviewer: ```suggestion
Adds line numbers to output in the [Pretty](../../interfaces/formats.md#pretty) format.
```
